gravitee-apim-console-webui/src/management/api/component/gio-connector-dialog/gio-connector-dialog.component.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 import { Component, Inject } from '@angular/core';
-import { MatLegacyDialogRef as MatDialogRef, MAT_LEGACY_DIALOG_DATA as MAT_DIALOG_DATA } from '@angular/material/legacy-dialog';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
```

Comment:
we broken the design here I think. Maybe we just need to add some padding in the dialog. ( I'm thinking about the button at the right top corner )
<img width="700" alt="Capture d’écran 2024-02-15 à 16 06 17" src="https://github.com/gravitee-io/gravitee-api-management/assets/25704259/812adf72-10d7-488e-a47b-6fe0a5a7fd38">
<img width="700" alt="Capture d’écran 2024-02-15 à 16 06 41" src="https://github.com/gravitee-io/gravitee-api-management/assets/25704259/6529fb0a-420e-4420-b309-6a551185ef1c">

Reply:
I'll keep it that way. It's a similar display on all the dialogs and a subject to fix globally with a single component. IMHO